started/cross-compiling.md

```diff
@@ -61,8 +61,16 @@ fyne-cross allows to build binaries and create distribution packages for the fol
 
 #### Installation
 
+For Go installtions above Go.16 you can use this command:
```

Comment:
You can install `fyne-cross` using the following command (requires Go 1.16 or later):